wdl/gwas/regenie_sub.wdl

```diff
@@ -39,6 +46,7 @@ task step2 {
         --phenoFile ${cov_pheno} \
         --phenoColList ${sep="," phenolist} \
         --pred ${pred} \
+        --use-null-firth ${firth_list} \
```

Comment:
can you add --firth-se so that the se is computed from the firth p-value and not the  standard test

Reply:
That's in the json options: https://github.com/Lipastomies/regenie-pipelines/blob/null_firth/wdl/gwas/regenie.json#L17

Reply:
https://rgcgithub.github.io/regenie/overview/

"The p-value reported in regenie is based on a likelihood ratio test (LRT), and we use the Hessian of the log-likelihood without the penalty term to estimate the standard error (SE). This may cause an issue in meta-analyses with rare variants, as the effect size estimate and SE may not match with the LRT p-value. Hence, we added an option --firth-se to report a SE computed instead from the effect size estimate and the LRT p-value."

Reply:
ah.... we were just looking with Juha M why p-value computed form Regenie beta and se does not match their reported beta..... we'll investigate further. Something does not match